### ISSUE TITLE
Terms Dropped in Ungrouped Expressions

### ISSUE THREAD

**Issue by hakiobo** [cregus/expression-evaluator#1]
https://github.com/cregus/expression-evaluator/blob/13780acd8d643de9a29c3a6e98122fd09f1c981f/src/main/kotlin/pl/kremblewski/expressionevaluator/Tokenizer.kt#L67

Whenever you split based on an, if it splits into more than 2 components, all but the first 2 are discarded 

For example, "1+2+4" would evaluate to 3 instead of 7, and "2*3*5" would evaluate to 6 instead of 30

**Comment by hakiobo** [comment]
```kt
leftToken = parse(parts.dropLast(1).joinToString(operator.symbol.toString()), data),
rightToken = parse(parts.last(), data),
```
I believe this would work as a fix, as it would cause the groupings to be evaluated from left to right

**Comment by cregus** [comment]
It has been fixed in version 1.0.1. Thanks for this report!
